<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#msgBrowser{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border:1px;
            border-radius:20px; 
            padding:2px 4px;  
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#msgTextEdit{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border:1px;
            border-radius:20px; 
            padding:2px 4px;  
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="usrTblWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#usrTblWidget{
			color: rgb(255, 255, 255);
            
            border:1px;
			background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border-radius:20px; 
            padding:2px 4px;  
}</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
  </widget>
  <widget class="QFontComboBox" name="fontCbx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeCbx">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{border-color:rgb(170, 255, 255);}</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
			color: rgb(255, 255, 255); 
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(130,120,226), stop:0.3 rgb(120,130,230), stop:1 rgb(125,140,226));
            border:1px;  
            border-radius:5px; 
            padding:2px 4px;
}
QPushButton{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(166,164,208), stop:0.3 rgb(171,152,230), stop:1 rgb(152,140,220));
            border:1px;
            border-radius:5px; 
            padding:2px 4px;  
}
QPushButton:pressed{    
            color: rgb(255, 255, 255); 
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(240,156,121), stop:0.3 rgb(220,160,140), stop:1 rgb(230,140,120));  
            border:1px;  
            border-radius:5px;
            padding:2px 4px; 
        }</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
			color: rgb(255, 255, 255); 
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(130,120,226), stop:0.3 rgb(120,130,230), stop:1 rgb(125,140,226));
            border:1px;  
            border-radius:5px; 
            padding:2px 4px;
}
QPushButton{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(166,164,208), stop:0.3 rgb(171,152,230), stop:1 rgb(152,140,220));
            border:1px;
            border-radius:5px; 
            padding:2px 4px;  
}
QPushButton:pressed{    
            color: rgb(255, 255, 255); 
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(240,156,121), stop:0.3 rgb(220,160,140), stop:1 rgb(230,140,120));  
            border:1px;  
            border-radius:5px;
            padding:2px 4px; 
        }</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QToolButton" name="boldTBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#boldTBtn{border-image:url(:/new/prefix1/Picture/_bold.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="italicTBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>倾斜</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#italicTBtn{border-image:url(:/new/prefix1/Picture/_italic.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="underlineTBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#underlineTBtn{border-image:url(:/new/prefix1/Picture/_xiahuaxian.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="colorTBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>更改字体颜色</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#colorTBtn{border-image:url(:/new/prefix1/Picture/_changecolor.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="saveTBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>保存聊天记录</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#saveTBtn{border-image:url(:/new/prefix1/Picture/_save.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="clearTBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>清空聊天记录</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#clearTBtn{border-image:url(:/new/prefix1/Picture/_clear.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enjoy talking!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inviteEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#inviteEdit{
			color: rgb(255, 255, 255);
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90,160,170), stop:0.3 rgb(60,203,205), stop:1 rgb(20,225,240));
            border:1px;
            border-radius:4px; 
            padding:2px 4px;  
}</string>
    </property>
   </widget>
   <widget class="QToolButton" name="inviteButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#inviteButton{border-image:url(:/new/prefix1/Picture/search.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="newwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>681</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#newwidget{border-image:url(:/new/prefix1/Picture/sea3.jpg);}</string>
   </property>
  </widget>
  <zorder>newwidget</zorder>
  <zorder>msgBrowser</zorder>
  <zorder>msgTextEdit</zorder>
  <zorder>usrTblWidget</zorder>
  <zorder>fontCbx</zorder>
  <zorder>sizeCbx</zorder>
  <zorder>sendBtn</zorder>
  <zorder>exitBtn</zorder>
  <zorder>boldTBtn</zorder>
  <zorder>italicTBtn</zorder>
  <zorder>underlineTBtn</zorder>
  <zorder>colorTBtn</zorder>
  <zorder>saveTBtn</zorder>
  <zorder>clearTBtn</zorder>
  <zorder>widget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
